bot/bot.py

```diff
@@ -106,81 +43,37 @@ async def ping_services(self) -> None:
                 await asyncio.sleep(constants.URLs.connect_cooldown)
 
     @classmethod
-    def create(cls) -> "Bot":
+    async def create(cls, allowed_roles: list, intents: discord.Intents) -> "Bot":
```

Comment:
Even though I came up with this, I was never really happy with it. I think it'd make more sense to just do this stuff in `__init__` while still allowing the args to be overwritten when instantiating the bot.

The redis session can be created outside and passed in as an arg.